app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activity.MainActivity">
+
+    <!--<ProgressBar-->
+        <!--android:id="@+id/pbLoad"-->
+        <!--android:layout_width="50dp"-->
+        <!--android:layout_height="50dp"-->
+        <!--android:layout_marginStart="8dp"-->
+        <!--android:layout_marginEnd="8dp"-->
+        <!--android:foregroundGravity="center"-->
+        <!--app:layout_constraintEnd_toEndOf="parent"-->
+        <!--app:layout_constraintStart_toStartOf="parent"-->
+        <!--tools:layout_editor_absoluteY="10dp"-->
+        <!--android:visibility="gone"/>-->
+
+
+    <!--Internet check-->
+
+    <TextView
+        android:id="@+id/tvInetMessg"
+        android:text="No signal! \n Please check your internet connection"
+        android:gravity="center"
+        android:textStyle="bold"
+        android:textSize="20dp"
```

Comment:
Text size should in "sp", and should be defined in dimens.xml file or in separate style.

Reply:
ready